src/components/static/pages/AboutUs.tsx

```diff
@@ -2,116 +2,257 @@ import * as React from "react";
+                img: IzzyReal,
+                cartoon: Izzy
+            }, 
+            {
+                name: "Savoy Schuler",
+                role: "Manager of Communication/Branding/Media",
+                quote: "-When human beings are exposed to opportunities to create and innovate curiosity compels them to do so.",
+                bio: "Savoy is passionate about creating these opportunities for tomorrows youth and -in particular- those least likely to be otherwise exposed to science and technology. An entrepreneur, engineer, public speaker, and humanitarian, Savoy can be found anywhere from developing Windows UI, to kindling the fire of a new foundation or business, to managing Nuevo Foundation's communication, branding, and media or coaching a debate team. He plays Quidditch on Wednesdays (Beaters are best) and spends his evenings trying to teach a made-up sign language to his deaf cat.",
+                img: SavoyReal,
+                cartoon: Savoy
+            },
+            {
+                name: "Deliana Escobari",
+                role: "VP Web Development, Curriculum Designer, and Content Creator",
+                quote: "-Everyone has the potential for greatness, they just need the right tools to unlock that potential.",
+                bio: "Dee started volunteering with Nuevo Foundation soon after moving to the Pacific Northwest in 2018. Her passion for education and technology lead her to contribute in many areas. She created the Web Basics Curriculum and the cartoons you see on this page. She is helping with the website redesign and implementation as part of her larger goal to spread the Nuevo Foundation mission and values as far as possible.",
+                img: DeeReal,
+                cartoon: Dee
+            },
+            {
+                name: "Aaron Malveaux",
+                role: "Manager of Communication/Branding/Media",
+                quote: "-\"Everybody is a genius. But if you judge a fish by its ability to climb a tree, it will live its whole life believing that it is stupid.\" - Albert Einstein",
+                bio: "Aaron is the most charismatic team member with broad experience in data privacy. He always ensures that every student is having a great time while learning new computer science concepts. Aaron works on reaching out to potential partners at schools and other nonprofit organizations",
+                img: AaronReal,
+                cartoon: Aaron
+            }
+        ];
+
+        return Team.map(person => {
```

Comment:
Let's set the `key` prop on the array items otherwise we get a console warning `Warning: Each child in a list should have a unique "key" prop.`

We can pass the `index` parameter to the callback function and use these as the keys.